air/src/options.rs

```diff
@@ -406,6 +427,55 @@ impl Default for PartitionOptions {
     }
 }
 
+// BATCHING OPTION
```

Comment:
nit: I would change this to "BATCHING METHOD".

Reply:
fixed